css/utilities/flex.css

```diff
@@ -0,0 +1,31 @@
+.align-i-ctr {
```

Comment:
do you think we can just say `flex-align-center` or just `align-center`. Similarly - `flex-align-self` or `align-self` 
and `flex-column`. 

Similar to what you did for `flex-wrap`. Just a suggestion. See what's easier to understand.

Reply:
I think we can not use `flex-align-center` or `align-center` as we have `align-content`, `align-items`, and `align-self` property. So differentiate b/w them, I am doing it like this.

And I am using `align-i-ctr`, instead of `align-items-center`, just to short the name.